grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -226,12 +226,13 @@ class DocumentTemplateService {
         def requisitionItems = requisitionInstance?.requisitionItems?.collect { RequisitionItem requisitionItem ->
             def demand
             def quantityOnHand
-            if (requisitionInstance?.destination?.isWard()) {
-                // if request FROM Ward then pull demand from origin to ward and use quantity counted as quantity on hand
+            if (requisitionInstance?.destination?.hasNoManagedInventoryAndSupportsSubmittingRequests()) {
+                // if request FROM "Ward" (location without managed inventory but supporting submitting requests),
+                // then pull demand from origin to that location
```

Comment:
Let's no use "ward" because it could be all sorts of location types. Let's call it a downstream consumer. In fact, I think that's what I want the method to be called as well. Either of these sounds good to me.
```
Boolean isDownstreamConsumer() 
Boolean isDownstreamEndpoint()
```